### PR TITLE
Feat: Rest explorer revamp priority 3, priority 4

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
Rest explorer revamp priority 3, priority 4 
Fixes issue #737 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@app/pages/Dashboard/Dashboard.ViewModel.old.ts [hunk 1]
```diff
@@ -23,6 +23,10 @@ export class DashboardViewModel {
     await this.teamRepository.setOpenTeam(teamId);
   };
 
+  public getActiveWorkspace = () => {
+    return this.workspaceRepository.getActiveWorkspace();
+  };
```
Comment (Astitva877) [inline src/packages/@app/pages/Dashboard/Dashboard.ViewModel.old.ts:28]: This is old view model, please add this is in new View model.

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@app/pages/Dashboard/Dashboard.ViewModel.old.ts (hunk 1 above)
Comment (itsmdasifraza) [reply src/packages/@app/pages/Dashboard/Dashboard.ViewModel.old.ts]: resolved

**Review by Astitva877 — APPROVED** [review]

**Review by gc-codes — APPROVED** [review]

**Pull request merged**